core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -51,17 +51,19 @@ object BattleDamage {
                 }
             }
 
-            var adjacentUnitBonus = 0
+            val adjacentUnits = combatant.getTile().neighbors.flatMap { it.getUnits() }
+            
             for (unique in civInfo.getMatchingUniques("+[]% Strength for [] units which have another [] unit in an adjacent tile")) {
                 if (combatant.matchesCategory(unique.params[1])
-                    && combatant.getTile().neighbors.flatMap { it.getUnits() }
-                    .any { it.civInfo == civInfo && it.matchesFilter(unique.params[2]) } 
+                    && adjacentUnits.any { it.civInfo == civInfo && it.matchesFilter(unique.params[2]) } 
                 ) {
-                    adjacentUnitBonus += unique.params[0].toInt()
+                    modifiers.add("Adjacent units", unique.params[0].toInt())
                 }
             }
-            if (adjacentUnitBonus != 0)
-                modifiers["Adjacent unit"] = adjacentUnitBonus
+            
+            for (unique in adjacentUnits.flatMap { it.getMatchingUniques("[]% Strength for [] units in adjacent tiles you are at war with") })
```

Comment:
Do you think 'you are at war with' is more clear-cut than 'enemy'?

Reply:
The main problem I found when doing these kind of things, is that the order of adjectives doesn't line up.
For example when using "[]% Strength for enemy [] units" you also have "[]% Strength for enemy All units" which doesn't sound right, but if you use "[]% Strength for [] enemy units" you also have "[]% Strength for land enemy units" which also doesn't sound right. Moving the enemy to a different spot solves this, but does make it longer.

For now I've changed it to "[]% Strength for all enemy [] units in adjacent [tileFilter] tiles", which has the same problem but with the tileFilter, but that is the way it is.

Reply:
Sounds thought-out :)